server/src/test/java/umm3601/user/UserControllerSpec.java

```diff
@@ -31,20 +32,23 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import io.javalin.core.JavalinConfig;
+import io.javalin.core.validation.ValidationException;
 import io.javalin.http.BadRequestResponse;
 import io.javalin.http.Context;
+import io.javalin.http.HandlerType;
 import io.javalin.http.NotFoundResponse;
 import io.javalin.http.util.ContextUtil;
-import io.javalin.plugin.json.JavalinJson;
-
+import io.javalin.plugin.json.JavalinJackson;
+import io.javalin.plugin.json.JsonMapperKt;
```

Comment:
You can directly import the `JSON_MAPPER_KEY` property instead of the entire class
```suggestion
import static io.javalin.plugin.json.JsonMapperKt.JSON_MAPPER_KEY;
```

Then replace `JsonMapperKt.JSON_MAPPER_KEY` with `JSON_MAPPER_KEY` where it's used.